examples/prostate/configs/prostate_central/config/config_fed_client.json

```diff
@@ -0,0 +1,31 @@
+{
+  "format_version": 2,
+
+  "executors": [
+    {
+      "tasks": ["train", "submit_model"],
+      "executor": {
+        "path": "pt.trainers.prostate_trainer.ProstateTrainer",
+        "args": {
+          "aggregation_epochs": 10,
+          "train_config_filename": "config/config_train.json"
```

Comment:
Change the value of "train_config_filename" to simply "config_train.json" (remove "config/"). Also change the code of config loading to use Workspace API, which manages the app folder structure.